wiki/en/Installation-for-iOS

```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: "Installation for iOS"
+lang: "en"
+permalink: "/wiki/Installation-for-iOS"
+---
+
+{% include breadcrumb.html root="Using Jamulus" branch1="Getting Started" branch1-url="Getting-Started" %}
+
+
+# Installation for iOS
+
+Make sure you've already read the [Getting Started](Getting-Started) page.
+
+## Things to note about iOS
+Although Jamulus on iOS works quite well on new devices if everything is set up correctly, we strongly recommend only doing so if you have an audio interface and an USB-Ethernet adapter. To connect both devices to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered USB-hub).
```

Comment:
```suggestion
Jamulus on iOS works quite well on new devices if everything is set up correctly. However, we strongly recommend you use an audio interface and an USB-Ethernet adapter. To connect these to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered USB-hub).
```

I wasn't sure what "only doing so" referred to. Also slightly confusing use of "device".